yt/frontends/boxlib/fields.py

```diff
@@ -523,15 +525,24 @@ def setup_fluid_fields(self):
 
 
 def _nice_species_name(field):
-    spec_match = spec_finder.search(field)
-    nice_name = "".join(spec_match.groups())
+    try:
+        spec_match = spec_finder.search(field)
+        nice_name = "".join(spec_match.groups())
+
+    except AttributeError as e:
+        # Catch exception cases for e.g. molecules (H2O, C6H6)
+        print("Could not parse species ", field)
+        nice_name = tex_label = field
+        return nice_name, tex_label
+
     # if the species field is a descriptive name, then the match
     # on the integer will be blank
     # modify the tex string in this case to remove spurious tex spacing
     lab = r"X\left(^{%s}%s\right)"
     if spec_match.groups()[-1] == "":
         lab = r"X\left(%s%s\right)"
     tex_label = lab % spec_match.groups()[::-1]
+
```

Comment:
Given the existence of #4845, where this file is moved, we're going to get merge conflict anyway so let's make their resolution as easy as possible by keeping the diff minimal.
```suggestion
```

Reply:
Just to make sure I understand, you are only suggesting to remove the blank line here?

Reply:
I think #4845 is ready to merge, right?
should we wait on anything else in that PR?

Reply:
> Just to make sure I understand, you are only suggesting to remove the blank line here?

Yes

Reply:
> should we wait on anything else in that PR?

I just want to honor the "one week or so" delay I promised in my last call for objections. Reviews came in faster than I anticipated, but there's no harm in waiting a couple days more, I think.